main.py

```diff
@@ -1,10 +1,51 @@
 import init_django_orm  # noqa: F401
 
+import json
+
 from db.models import Race, Skill, Player, Guild
 
 
+def create_race(race: dict) -> Race | bool:
+    return Race.objects.get_or_create(
+        name=race["name"],
+        description=race["description"]
+    )
+
+
+def create_skill(skills: list[dict], race: Race) -> None:
+    for skill in skills:
+        Skill.objects.get_or_create(
+            name=skill["name"],
+            bonus=skill["bonus"],
+            race=race
+        )
+
+
+def create_guild(guild: dict) -> Guild | None:
+    if not guild:
+        return
+    guild_obj, created = Guild.objects.get_or_create(
```

Comment:
You could omit using `created` variable by replacing it with `_`
```suggestion
    guild_obj, _ = Guild.objects.get_or_create(
```